helm/fiftyone-teams-app/values.yaml

```
@@ -421,6 +421,14 @@ teamsAppSettings:
     repository: voxel51/fiftyone-teams-app
     # -- Image tag for teams-app. Defaults to the chart version.
     tag: ""
+
+  # -- Overrides the `FIFTYONE_API_URI` environment variable when set
```

```suggestion
  # -- Overrides the `FIFTYONE_API_URI` environment variable. When set
```

This prevents a run on sentence in the README.md:

```txt
Overrides the `FIFTYONE_API_URI` environment variable when set `FIFTYONE_API_URI` controls the value shown in the API Key Modal providing guidance for connecting to the FiftyOne Teams API. `FIFTYONE_API_URI` uses the value from apiSettings.dnsName if it is set, or uses the teamsAppSettings.dnsName
```


We may need to run the pre-commit hooks to get this to update the README.md. : )

---

README rebuilt